xbmc/video/VideoDatabase.h

```diff
@@ -83,6 +83,9 @@ namespace VIDEO
 #define VIDEODB_DETAILS_MOVIE_DATEADDED         VIDEODB_MAX_COLUMNS + 10
 #define VIDEODB_DETAILS_MOVIE_RESUME_TIME       VIDEODB_MAX_COLUMNS + 11
 #define VIDEODB_DETAILS_MOVIE_TOTAL_TIME        VIDEODB_MAX_COLUMNS + 12
+#define VIDEODB_DETAILS_MOVIE_RATING            VIDEODB_MAX_COLUMNS + 13
+#define VIDEODB_DETAILS_MOVIE_VOTES             VIDEODB_MAX_COLUMNS + 14
+#define VIDEODB_DETAILS_MOVIE_DEFAULT_RATING    VIDEODB_MAX_COLUMNS + 15
```

Comment:
I changed because in the view it changes the meaning. It's not the rating type anymore but it's the default rating type preferred for the media. I'll change it to VIDEODB_DETAILS_MOVIE_DEFAULT_RATING_TYPE